lib/zold/version_file.rb

```diff
@@ -0,0 +1,32 @@
+require 'semantic'
+
+module Zold
+  # Read and write .zoldata/version.
+  class VersionFile
+    def initialize(path)
+      @path = path
+    end
+
+    def save(version)
+      File.open(@path, 'w') do |file|
```

Comment:
@botanicus why not just `File.write()`? Or better `AtomicFile.write()` (we have in our code base).